src/main/scala/com/supersonic/datadog/datadog_schemas.scala

```diff
@@ -67,6 +69,24 @@ object Graph {
     case class Op(name: String)
   }
 
+  sealed trait EventOverlay {
+    def render :String = this match {
+      case EventTag(name, value, eventName) =>
+        val renderedValue = value.map(v => s":$v").getOrElse("")
+        s"$name$renderedValue + $eventName"
+      case EventFromTemplateVariable(value, eventName) => "$" + value.name + eventName
```

Comment:
You can use string interpolation, here and below

Reply:
Done